airflow/www/static/js/gantt.js

```diff
@@ -157,9 +164,19 @@ d3.gantt = () => {
         if (!(a.end_date instanceof moment)) {
           a.end_date = moment(a.end_date);
         }
+        if (a.queued_dttm) {
```

Comment:
```suggestion
        if (a.queued_dttm && !(a.queued_dttm instanceof moment)) {
```
You can combine the if statements

Reply:
will do